qiskit/transpiler/passes/routing/bip_mapping.py

```diff
@@ -166,6 +167,7 @@ def run(self, dag):
                 "BIPMapping requires the number of virtual and physical qubits to be the same. "
                 "Supply 'qubit_subset' to specify physical qubits to use."
             )
+        disjoint_utils.check_layout_isolated_to_component(dag, self.coupling_map)
```

Comment:
Not introduced here, but seeing this used, I wonder if a better name could be `require_layout_isolated_to_component`. To me, it's a little less obvious that the method as named right now will throw an exception when the check fails.

Reply:
Sure, I'll rename it. I'm not attached to a particular name

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9840/commits/6ae66340ec32b68bec1ddd95be3ad9dcb3c79de7